cmd/internal/planetscale_edge_database.go

```diff
@@ -488,6 +506,17 @@ func buildVStreamRequest(tabletType psdbconnect.TabletType, table string, shard
 			}},
 		},
 	}
+
+	if lastKnownPk != nil {
```

Comment:
This fixes a bug: if `req.Vgtid.ShardGtids[0].TablePKs` is non nil, then copy phase is always started. Make sure to leave it nil if we're NOT intending to start a copy phase (lastKnownPk is nil)